### PR TITLE
Add `Writing Appfile` based on capability references docs

### PR DESCRIPTION
**Pull request by zzxwill**
Introduced `vela show` with and without flag `--no-website`, and
official site to help setting properties for workload types and
traits.

### CODE REVIEW

File: docs/en/developers/writing-appfile.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        replicas: 4
+        timezone: "America/Los_Angeles"
+```
+
+## Look up the reference doc in a [kubevela.io](https://kubevela.io/#/en/developers/references/)
+
+> Platform team extended Workload type or Trait currently could not display in KubeVela official site. 
+
+If you just want to look up KubeVela built-in Workload types and Traits, directly visiting the official site [https://kubevela.io/#/en/developers/references/](./references/README.md)
+is also recommended.
```
Comment (wonderflow) [inline docs/en/developers/writing-appfile.md:115]: 这个要么放在最前面，要么放在最后面。中间不太好。

File: docs/en/developers/writing-appfile.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+## Show up the properties reference on the Cli console
+
+If you prefer to show the properties reference on the Cli console, just add flag `--no-website` to command `vela show`.
```
Comment (wonderflow) [inline docs/en/developers/writing-appfile.md:119]:
```suggestion
If you can't invoke a browser and just want to see the properties reference on the terminal console, just add flag `--no-website` to command `vela show`.
```

File: docs/en/developers/set-autoscale.md [hunk 3]
```diff
@@ -29,6 +29,8 @@ Introduce how to automatically scale workloads by cron.
           timezone: "America/Los_Angeles"
   ```
 
+> The full specification of `autoscale` could show up by `vela show autoscale` or be found on [kubevela.io](references/traits/autoscale.md)
```
Comment (wonderflow) [inline docs/en/developers/set-autoscale.md:32]: 这些是自动生成的吗？

**Review by wonderflow — APPROVED** [review]
LGTM

File: docs/en/developers/set-autoscale.md (hunk 3 above)
Comment (zzxwill) [reply docs/en/developers/set-autoscale.md:32]: 我手动加的。

File: docs/en/developers/writing-appfile.md (hunk 1 above)
Comment (zzxwill) [reply docs/en/developers/writing-appfile.md:115]: Okay.

**Comment by wonderflow** [comment]
The name `--no-website` is boring, I suggest `--raw` if no better name.

File: docs/en/developers/writing-appfile.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        replicas: 4
+        timezone: "America/Los_Angeles"
+```
+
+## Show up the properties reference on the Cli console
+
+If you can't invoke a browser and just want to see the properties reference on the terminal console, just add flag `--no-website` to command `vela show`.
+
+```shell
+$ vela show webservice --no-website
```
Comment (resouer) [inline docs/en/developers/writing-appfile.md:115]:
```suggestion
$ vela show webservice --terminal
```

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**